completions/fish/crio.fish

```diff
@@ -56,14 +56,14 @@ complete -c crio -n '__fish_crio_no_subcommand' -f -l profile -d 'enable pprof r
 complete -c crio -n '__fish_crio_no_subcommand' -f -l profile-port -r -d 'port for the pprof profiler'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l read-only -d 'setup all unprivileged containers to run as read-only (default: false)'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l registry -r -d 'registry to be prepended when pulling unqualified images, can be specified multiple times (default: configured in /etc/containers/registries.conf)'
-complete -c crio -n '__fish_crio_no_subcommand' -l root -s r -r -d 'crio root dir (default: "/home/sascha/.local/share/containers/storage")'
```

Comment:
It was a bit egoistic to think that every Linux user has my name. :innocent:

Reply:
Probably a safe bet that they don't.  ...